ui/sse_server.py

```
@@ -0,0 +1,367 @@
+                    self.end_headers()
+                    self.wfile.write(b"Forbidden: origin not allowed")
+                    return None
+                if _SSE_TOKEN:
+                    token = self.headers.get("X-DSS-Token", "")
+                    # Fallback to query parameter for browser EventSource clients
+                    if not token:
+                        parsed = urlparse(self.path)
+                        query_params = parse_qs(parsed.query)
+                        token = query_params.get("token", [""])[0] or query_params.get("sse_token", [""])[0]
+                    if token != _SSE_TOKEN:
+                        print(f"[DSS SSE] Token rejected")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)